src/Microsoft.CodeAnalysis.Analyzers/Core/CodeAnalysisDiagnosticsResources.resx

```diff
@@ -503,4 +503,31 @@
   <data name="EnableAnalyzerReleaseTrackingRuleDescription" xml:space="preserve">
     <value>Enabling release tracking for analyzer packages helps in tracking and documenting the analyzer diagnostics that ship and/or change with each analyzer release. See details at https://github.com/dotnet/roslyn-analyzers/blob/master/src/Microsoft.CodeAnalysis.Analyzers/ReleaseTrackingAnalyzers.Help.md.</value>
   </data>
+  <data name="DefineDiagnosticTitleCorrectlyDescription" xml:space="preserve">
+    <value>The diagnostic title is recommended to be a short sentence without any period ('.)' nor any line return ('\r', '\n' or '\r\n').</value>
+  </data>
+  <data name="DefineDiagnosticTitleCorrectlyMessage" xml:space="preserve">
+    <value>Define diagnostic title correctly</value>
```

Comment:
Can we make the message more specific, such as `The diagnostic title should not contain a period or any line return character`?